lib/Table/TableStyleMap.ts

```diff
@@ -97,52 +101,53 @@ export default class TableStyleMap<T extends TableStyleMapSymbolTraits> {
     // If column type is `scalar` and binStyles
     if (
       (this.traitValues.mapType === "bin" || !this.traitValues.mapType) &&
-      this.column?.type === TableColumnType.scalar &&
       this.traitValues.bin &&
       this.traitValues.bin.length > 0
     ) {
+      const mapValueToStyle: MapValueToStyle<T> = (value) => {
+        if (typeof value !== "number") {
+          return this.traitValues.null;
+        }
+
+        const binStyles = this.traitValues.bin ?? [];
+        let i;
+        for (
```

Comment:
I think a one line comment explaining what the loop computes would be nice.

Not sure if the continue in the body adds anything.